documentation/samplefiles-dpg/samplereadme.md

```diff
@@ -0,0 +1,80 @@
+# [[ServiceName]]
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for [[ServiceName]].
+
+## Getting Started
+
+To build the SDKs for My API, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+
+> `autorest readme.md`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the [[ServiceName]].
+
+```yaml
+openapi-type: [[OpenApiType]]
+tag: package-[[Version]]
+```
+
+### Security Configuration
+
+These are the global settings for security configuration
+
+> see https://github.com/Azure/autorest/blob/main/docs/generate/authentication.md
+(If authentication is apikey, then the following configuration shall be `security: AzureKey` and `security-header-name: [[HeaderName]]`)
+
+```yaml
+security: AADToken
+security-scopes: [[SecurityScopes]]
+```
+
+
```

Comment:
We are in the process of moving all the security configuration to the REST API doc, so soon this section can be removed.

Reply:
sure. When you have finished moving all the security configuration to REST API doc, please let we know, and we will remove it.